### PR TITLE
Overall reverses lavaland powercreep

### PR DESCRIPTION
**Pull request by mystery3525**
## About The Pull Request
This overall reverses the powercreep present in lavaland where very tanky mobs require very tanky weaponry to fight by bringing miners down to somewhat-reasonable amounts of possible damage per second able to be given and also brings down the typical lavaland mobs health as well. As a consequence, reverts https://github.com/BeeStation/BeeStation-Hornet/pull/7552

<details>
<summary>Path Changes</summary>

```
/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient
/mob/living/simple_animal/hostile/asteroid/goliath
/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/magmawing
/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/icewing
/mob/living/simple_animal/hostile/asteroid/basilisk
/mob/living/simple_animal/hostile/asteroid/hivelord/legion/dwarf
/mob/living/simple_animal/hostile/asteroid/hivelord/legion/advanced
/mob/living/simple_animal/hostile/asteroid/hivelord/legion
/mob/living/simple_animal/hostile/asteroid/hivelord
/mob/living/simple_animal/hostile/asteroid/goldgrub
/mob/living/simple_animal/hostile/megafauna
/mob/living/simple_animal/hostile/megafauna/drake
/mob/living/simple_animal/hostile/megafauna/colossus
/mob/living/simple_animal/hostile/megafauna/bubblegum
/mob/living/simple_animal/hostile/megafauna/blood_drunk_miner
/mob/living/simple_animal/hostile/megafauna/hierophant
/mob/living/simple_animal/hostile/megafauna/legion
/mob/living/simple_animal/hostile/megafauna/swarmer_swarm_beacon
/obj/structure/spawner/lavaland
/obj/structure/geyser (set max_integrity to 150)
```
</details>

PKA: 40 per hit -> 20 per hit
PKC: 50 per bubble pop -> 25 per bubble pop (holy cow)

For comparison, the laser gun does 20 burn damage on a successful hit.

This will by no means make the typical miner's job easier or harder, it will just address miners fighting crew head-on and maybe give them a fighting chance. Also, the occasional ash drake on the station will be 50% easier to fight (probably).

## Why It's Good For The Game

Powerfulbacon (a not-well-known contributor for this codebase) suggested it, additionally addresses https://github.com/BeeStation/BeeStation-Hornet/issues/8010

Additional Note: If the maintainers want this to go more than 1/2 overall damage we can try out lower amounts.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

damage to naked ashwalker with stock PKA lavaland atmos (4 shots to crit):

https://user-images.githubusercontent.com/30960302/202105790-b60caefd-6855-42c5-8713-1fe64e4517e3.mp4

damage to naked ashwalker with stock PKA weakened atmos (10 shots to crit):

https://user-images.githubusercontent.com/30960302/202106474-8e5ebf6a-4484-4725-97d9-64ac7bfac2f9.mp4

 

</details>

## Changelog
:cl:
balance: halved lavaland mobs health
balance: halved PKA and PKC damage
balance: Removed minimum pressure requirement from PKA and PKC, so they do the same amount of damage in vacuums again.
/:cl:

### CODE REVIEW

**Comment by mystery3525** [comment]
I have just been informed that as a consequence, this will buff any (non-miner) inhabitant of lavaland

**Comment by CydiaLamiales** [comment]
This does nerf the effectiveness of PKAs as an antag weapon, considering the special modkit that miners can buy from the uplink

**Comment by hotimep** [comment]
> I have just been informed that as a consequence, this will buff any (non-miner) inhabitant of lavaland

ASHLIZARD MAINS RISE UP!!!!

File: code/modules/mining/equipment/kinetic_crusher.dm [hunk 1]
```diff
@@ -25,13 +25,13 @@
...
-	var/detonation_damage = 50
-	var/backstab_bonus = 30
+	var/detonation_damage = 25
+	var/backstab_bonus = 15
 
 /obj/item/kinetic_crusher/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/butchering, 60, 110) //technically it's huge and bulky, but this provides an incentive to use it
-	AddComponent(/datum/component/two_handed, force_unwielded=0, force_wielded=20)
+	AddComponent(/datum/component/two_handed, force_unwielded=0, force_wielded=10)
```
Comment (PowerfulBacon) [inline code/modules/mining/equipment/kinetic_crusher.dm]: I think this is too bad, considering a toolbox does 12 if I'm not mistaken. Perhaps bringing it up to 14 (or any value higher than the knife) could be a good change, especially considering how much worse melee is against simplemobs

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -2230,11 +2230,11 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 
 /datum/uplink_item/role_restricted/pressure_mod
 	name = "Kinetic Accelerator Pressure Mod"
-	desc = "A modification kit which allows Kinetic Accelerators to do greatly increased damage while indoors and in a vacuum. \
+	desc = "A modification kit which allows Kinetic Accelerators to do greatly increased damage while indoors. \
 			Occupies 35% mod capacity."
 	item = /obj/item/borg/upgrade/modkit/indoors
-	cost = 5 //you need two for full damage, so total of 10 for maximum damage
-	limited_stock = 2 //you can't use more than two!
+	cost = 10 //you need one for full damage, so total of 10 for maximum damage
```
Comment (PowerfulBacon) [inline code/modules/uplink/uplink_items.dm]: I don't see anywhere where you buffed the pressure reduction module.
If we are nerfing through KPA to do reasonable damage in space, then wouldn't this now be worth less as opposed to more? Compare this to other items of similar value

Comment (mystery3525) [reply code/modules/uplink/uplink_items.dm]: It'll only be worth for indoors, would 5 TC be okay?

**Comment by PowerfulBacon** [comment]
> balance: removed low-pressure damage modifier

This is incorrect

**Comment by mystery3525** [comment]
shit yeah by bad that's fixed now

**Comment by PowerfulBacon** [comment]
I thought it was refering to the pKA pressure attachment

**Review by PowerfulBacon — APPROVED** [review]

**Comment by zeskorion** [comment]
good change. lavaland feels artificially out of line with the base game

**Comment by zeskorion** [comment]
next up: makw megafauna balanced if it gets on-station

**Comment by Rukofamicom** [comment]
Megafauna should lose HP gradually, only use melee attacks and drop no loot if they're off Z-level. The necropolis cannot sustain them if they are too far away.

**Comment by PowerfulBacon** [comment]
Lets see how this affects the game.

**Pull request merged**